perseo-main/src/main/java/com/telefonica/iot/perseo/LogLevelServlet.java

```diff
@@ -65,21 +66,28 @@
     @Override
     protected void doPut(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        String levelName = request.getParameter("level");
-        logger.info("changing log level to " + levelName);
-        Level level = levels.get(levelName);
-        if (level == null) {
-            logger.error("invalid log level: " + levelName);
-            response.setCharacterEncoding("UTF-8");
-            response.setContentType("application/json");
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            response.getOutputStream().print("{\"errorMessage\":\"invalid log level\"}");
-            return;
+        try {
+        	String levelName = request.getParameter("level");
+            logger.info(String.format("changing log level to %s",levelName));
+            Level level = levels.get(levelName);
+            if (level == null) {
+                logger.error(String.format("invalid log level: %s",levelName));
+                response.setCharacterEncoding("UTF-8");
+                response.setContentType("application/json");
+                response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                response.getOutputStream().print("{\"errorMessage\":\"invalid log level\"}");
+                return;
+            }
+            synchronized (mutex) {
+                LogManager.getRootLogger().setLevel(level);
+            }
+            response.setStatus(HttpServletResponse.SC_OK);      
+        }catch(IOException e) {
```

Comment:
Wrong code style

* A whitespace should be inserted between `}` and `catch`.
* A whitespace should be inserted between `catch` and `(`

Please review more `}catch` and `catch(` in the overall contribution. This is not the only place in which I have seen thinks like this.

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673